server.go

```diff
@@ -58,9 +58,15 @@ func constructContainerInfo(r *http.Request) containerInfo {
 	if len(r.URL.Query()["port"]) > 0 {
 		c.port = r.URL.Query()["port"][0]
 	}
+
 	if len(r.URL.Query()["image"]) > 0 {
 		c.dockerImage = r.URL.Query()["image"][0]
 	}
+
+	if len(r.URL.Query()["profile"]) > 0 {
+		c.dockerProfile = dockerProfile(r.URL.Query()["profile"][0])
```

Comment:
Could we do some validation of the "profile" parameter, if the user gives something else than `defaultDockerProfile` or `weakDockerProfile`? At the moment, it does not seem problematic, but it will be more difficult to assess when we add more parameters & profiles.